proto/dwtools/abase/l0.test/Long.test.s

```diff
@@ -11814,56 +11814,157 @@ function longAreRepeatedProbe( test )
 function longAllAreRepeated( test )
 {
 
+  test.case = 'empty array';
   var got = _.longAllAreRepeated( [] );
-  test.identical( got, true );
+  var expected = true;
+  test.identical( got, expected );
 
+  test.case = 'identical numbers of array';
   var got = _.longAllAreRepeated( [ 1, 1 ] );
-  test.identical( got, true );
+  var expected = true;
+  test.identical( got, expected );
 
+  test.case = 'single number of array';
   var got = _.longAllAreRepeated( [ 1 ] );
-  test.identical( got, false );
+  var expected = false;
+  test.identical( got, expected );
 
+  test.case = 'some numbers of array are identical';
   var got = _.longAllAreRepeated( [ 1, 2, 2 ] );
-  test.identical( got, false );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
 
+  test.case = 'identical strings of array';
+  var got = _.longAllAreRepeated( [ 'string', 'string' ] );
```

Comment:
Please, use complex data types as elements of srcLong. For example, use arrays and maps
```js
var src = [ [ 1 ], [ 1 ], [ 2 ] ];
/* */
var src = [ { a : 1 }, { a : 2 }, { a : 1 } ];
```
See examples in file `Array.test.s`, routine `arrayRemovedElement_` and other test routines.